code/modules/mob/living/carbon/xenomorph/hive_status.dm

```diff
@@ -1285,6 +1284,10 @@
 	xeno_message(SPAN_XENOANNOUNCE("You sense that [english_list(defectors)] turned their backs against their sisters and the Queen in favor of their slavemasters!"), 3, hivenumber)
 	defectors.Cut()
 
+/datum/hive_status/proc/override_evilution(evil, override)
+	if(SSxevolution)
+		SSxevolution.override_power(hivenumber, evil, override)
```

Comment:
Is it possible for this subsystem to not be present?

I mean the check is okay, but it seems unnecessary? I'm more concerned what situation won't have the subsystem?

Reply:
I just copypasted it from another place, not sure